charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/clusterrole.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+  name: {{ template "common.naming.fullname" . }}-apiservice
   annotations:
     "helm.sh/hook": pre-install,pre-upgrade,post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
   labels:
-    app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 4 }}
+    app: {{ template "common.naming.name" . }}
```

Comment:
Should we keep the `app` label? `common.labels` already includes `app.kubernetes.io/name: {{ include "common.naming.name" . }}`